packages/iris-grid/src/AdvancedFilterCreator.tsx

```diff
@@ -102,6 +103,14 @@ class AdvancedFilterCreator extends PureComponent<
     return { key: shortid() };
   }
 
+  static assertFilterOperatorValue(
```

Comment:
In terms of code structure, this would make more sense in `FilterOperator.ts` here instead of as a static function on AdvancedFilterCreator.